src/activity-logger/background/index.js

```diff
@@ -9,3 +9,68 @@ browser.webNavigation.onCommitted.addListener(({url, tabId, frameId}) => {
     maybeLogPageVisit({url, tabId})
 
 })
+
+
+//********************************************************************************/
+var timestamp = new Date().getTime;
+
+var number_of_opened_tabs = 0;
+
+var tabList = [];
+
+
+/*chrome.runtime.onMessage.addListener(function(response,sender,sendResponse){
+    alert(response);
+});*/
+
+
+browser.tabs.onCreated.addListener(function (tab){
+    number_of_opened_tabs = number_of_opened_tabs + 1;
+    //console.log('total opened tabs: ' + number_of_opened_tabs);
+    var t = {
+        id : tab.id,
+        activetime : 0,
+        isActive : false,
+        lastActivatedtime : Number(new Date().getTime())
```

Comment:
I used `new Date().getTime()` so that we can easily get the number of milliseconds that a particular tab is activated. Is it a good choice?